packages/flutter/lib/src/widgets/tween_animation_builder.dart

```diff
@@ -212,3 +216,175 @@ class _TweenAnimationBuilderState<T extends Object?>
     return widget.builder(context, _currentTween!.evaluate(animation), widget.child);
   }
 }
+
+/// Widget that animates an [Animatable] value and repeats indefinitely.
+///
+/// The animation continuously cycles from the [Animatable] evaluated at 0.0 to
+/// the value at 1.0.
```

Comment:
I'd like to add a bit more detail to this paragraph. (The last sentence assumed the removal of `reverse`; feel free to tweak it otherwise.)
```dart
/// The animation continuously cycles from the value produced by the
/// [Animatable] at 0.0 to the value at 1.0. The [builder] receives the
/// current value produced by this [Animatable] and builds the child from it.
/// When the animation reaches 1.0 it jumps immediately back to the value at
/// 0.0.
```

Reply:
With `reverse` or `reverseMode` I've rephrased it as follows:
```dart
/// The animation continuously cycles from the value produced by the
/// [Animatable] at 0.0 to the value at 1.0. The [builder] receives the
/// current value produced by this [Animatable] and builds the child from it.
/// When the animation reaches 1.0, it either restarts from 0.0 or reverses
/// direction, based on the [repeatMode].
```